### ISSUE TITLE
Play animation on mouse hover not working

### ISSUE THREAD

**Issue by ahrbil** [LottieFiles/lottie-react#47]
## Overview

The `Player` component doesn't play on hover when passing the `hover` prop 

## Consuming repo

> What repo were you working in when this issue occurred?

https://codesandbox.io/s/inspiring-antonelli-5xhk4

## Labels

- [x] Add the `Type: Bug` label to this issue.

Thank you 🙏

**Comment by karamalie** [comment]
fixed with version 3.3.0. Let me know if it works as expected

**Comment by ahrbil** [comment]
Thank you @karamalie 🙏 
Works perfectly in v 3.3.0

**Comment by ahrbil** [comment]
Hello @karamalie, sorry for reopening this issue
play on hover is working correctly in v 3.3.0
However, it breaks when passing `hover` with `keepLastFrame` prop as it plays only the last frame of the animation.
I think for this to be fixed we need to check if keepLastFrame is passed then we stop the animation (this will reset the animation to play from the first frame) before playing it again.

```js
this.container.addEventListener('mouseenter', () => {
        if (hover && this.state.playerState !== PlayerState.Playing) {
          // Reset the player before playing when keepLastFrame is passed
          if(this.props.keepLastFrame){
             this.stop();
          }

          this.play();
        }
});
```
Thank you 🙏

**Comment by karamalie** [comment]
fixed in 3.3.1 . you re right. after one loop it breaks. added your code to the mix. have a look :) Do close the issue if everythings ok! happy coding
